google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/v1/Publisher.java

```diff
@@ -655,8 +655,8 @@ public static Builder newBuilder(String topicName) {
 
     // Meaningful defaults.
     static final long DEFAULT_ELEMENT_COUNT_THRESHOLD = 100L;
-    static final long DEFAULT_REQUEST_BYTES_THRESHOLD = 1000L; // 1 kB
-    static final Duration DEFAULT_DELAY_THRESHOLD = Duration.ofMillis(1);
+    static final long DEFAULT_REQUEST_BYTES_THRESHOLD = 1L * 1024L * 1024L; // 1 MB
```

Comment:
Should be 1 * 1000 * 1000? 1 MB as shown in go/cps-libs-defaults

Python just updated the docstring to 1 MB here: https://github.com/googleapis/python-pubsub/pull/572